android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java

```diff
@@ -254,10 +259,10 @@ public static void openBuySendSwapActivity(
         }
     }
 
-    public static void openAssetDetailsActivity(
-            Activity activity, String chainId, BlockchainToken asset) {
-        assert activity != null;
-        Intent assetDetailIntent = new Intent(activity, AssetDetailActivity.class);
+    public static void openAssetDetailsActivity(@NonNull final Context context,
+            @NonNull final String chainId, @NonNull final BlockchainToken asset) {
```

Comment:
nit: we can get rid of `@NonNull final String chainI` as `BlockchainToken` has a chainId field.

Reply:
Perfect thanks, I'm changing it.